### ISSUE TITLE
Review arXiv PDF links

### ISSUE THREAD

**Issue by mpaillassa** [aboucaud/galcheat#104]
As discussed in #101, we decided not to have direct links to PDF for arXiv paper reference. Instead, we should prefer abstract links with the version in comments.

Now I am also wondering if it wouldn't be even better to have [ads links](https://ui.adsabs.harvard.edu/) ?

**Comment by aboucaud** [comment]
> Now I am also wondering if it wouldn't be even better to have [ads links](https://ui.adsabs.harvard.edu/) ?

Not sure, IMO the arxiv link is the most straightforward way to access the paper. Plus the link is written in plain text so the shorter the better and I'm worried about the unicode handling of a link such as 
https://ui.adsabs.harvard.edu/abs/2020A%26A...634A..48P/abstract

**Comment by ismael-mendoza** [comment]
I agree, let's stick to the arxiv links (without reference to a specific version in the URL)

**Comment by mpaillassa** [comment]
@ismael-mendoza @aboucaud Do we agree to use abstract arXiv links and versions mentioned in comments like:

```
sky_brightness:
    link: "https://arxiv.org/abs/0805.2366"
    comment: "(v5) See Table 2, first row."
```

Currently many arXiv references are still PDF links so I may make a PR to fix this.

**Comment by ismael-mendoza** [comment]
Thanks @mpaillassa for the ping on this. Yes I think that is the best option that we decided.
